extensions/amp-analytics/0.1/test/test-variables.js

```diff
@@ -214,15 +214,19 @@ describes.fakeWin('amp-analytics.VariableService', {amp: true}, (env) => {
     });
 
     it('handles array with no vars', () => {
-      check('${array}', 'foo,bar', {
-        'array': ['foo', 'bar'],
+      return check('${array}', 'foo,bar,3', {
+        'array': ['foo', 'bar', 3],
       });
     });
 
     it('handles empty var name', () => {
       return check('${}', '', {});
     });
 
+    it('handles null vars', () => {
+      return check('${arr}', ',notNull', {'arr': [null, 'notNull']});
```

Comment:
Just to confirm only `null` is handled differently? [null, undefined, 123, true, 'notNull'] returns ',undefined,123,true,notNull'?

Reply:
Undefined is also escaped early. Added a test for this too.